src/main/java/seedu/address/logic/commands/MarkCommand.java

```diff
@@ -67,7 +67,7 @@ private static Task createMarkedTask(Task task) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Task> lastShownList = model.getFilteredTaskList();
+        List<Task> lastShownList = model.getSortedTaskList();
```

Comment:
Is it safe to do this here? Since not all the commands display the result as a sorted list.

Reply:
I see. That makes sense. Maybe we can improve on this in the next iteration.